bootstrap.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 kubectl apply -f .infrastructure/mysql/ns.yml
+kubectl apply -f .infrastructure/app/ns.yml
```

Comment:
Ensure that the namespace configuration for the application is correct and necessary before applying it. This is crucial for resource isolation and management within Kubernetes.